src/js/modules/FormFeedback.js

```diff
@@ -0,0 +1,231 @@
+import React, { Component } from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import { postFeedback } from 'js/helpers/fetchData';
+import { logEvent } from 'js/helpers/googleAnalytics';
+import SectionHeaderSerif from 'js/modules/SectionHeaderSerif';
+
+import emojiDisappointed from 'images/emojis/disappointed.png';
+import emojiSad from 'images/emojis/sad.png';
+import emojiNeutral from 'images/emojis/neutral.png';
+import emojiSlightlySmiling from 'images/emojis/slightly_smiling.png';
+import emojiGrinning from 'images/emojis/grinning.png';
+
+const i18nMessages = defineMessages({
```

Comment:
I know we've gotten into a patter of including these defineMessage object literals inside the component. Just thinking about whether they should be imported from another i18nMessage dir or something instead? Could that help with shareability between components? No real constructive comment, just thinking out loud.